stable/nginx-ingress/templates/podsecuritypolicy.yaml

```diff
@@ -20,7 +20,11 @@ spec:
     #- 'projected'
     - 'secret'
     #- 'downwardAPI'
+  {{- if .Values.controller.hostNetwork }}
+  hostNetwork: true
```

Comment:
you can do 
````
hostNetwork: {{ .Values.controller.hostNetwork | default false }}
```

and then remove the if statment

Reply:
It should be `hostNetwork: {{ .Values.controller.hostNetwork }}`. The default value comes from `values.yaml`.